src/common/mfu_flist_copy.c

```diff
@@ -17,8 +17,11 @@
 #include <getopt.h>
 #include <time.h> /* asctime / localtime */
 #include <regex.h>
+
+#ifdef HAVE_LIBATTR
 #include <attr/libattr.h>
 #include <attr/error_context.h>
```

Comment:
I think we don't need this include anymore.

Reply:
Good idea.  I dropped it.